src/Model/Event.php

```diff
@@ -291,6 +291,9 @@ public static function store($arr)
 		}
 
 		$contact = DBA::selectFirst('contact', [], $conditions);
+		if (!DBA::isResult($contact)) {
+			Logger::warning('Contact not found', ['condition' => $conditions]);
```

Comment:
No exception throwing?

Reply:
I would like to see the condition. Means: There will still be the notices, but then we have got the log entry to have a deeper dive into it.